util/config_to_js.go

```diff
@@ -0,0 +1,26 @@
+package util
+
+import (
+	"io/ioutil"
+	"path/filepath"
+
+	"github.com/kiali/kiali/config"
+	"github.com/kiali/kiali/log"
+)
+
+// ConfigToJS generates env.js file from Kiali config
+func ConfigToJS() {
+	log.Info("Generating env.js from config")
+	path, _ := filepath.Abs("./console/env.js")
+
+	content := "window.WEB_ROOT='" + config.Get().Server.WebRoot + "';"
+
+	log.Info("------")
```

Comment:
1) I think this should be at debug level. We already log what the web root at info level in server/server.go line 43.
2) Can make it more clear what this is logging? People will see "----" surrounding the `content` value in the log file, but they won't have any idea what this is for. In other words, don't just log "-----" at the top and bottom, I would change those three log calls to be something like this:

```
log.Debugf("The content of %v will be:\n%v", path, content);
```